packages/expo-av/android/src/main/java/expo/modules/av/player/SimpleExoPlayerData.java

```diff
@@ -83,8 +83,8 @@ public void load(final Bundle status, final LoadCompletionListener loadCompletio
     mLoadCompletionListener = loadCompletionListener;
 
     final Context context = mAVModule.getContext();
-    final BandwidthMeter bandwidthMeter = new DefaultBandwidthMeter.Builder(context).build();
-    final TrackSelector trackSelector = new DefaultTrackSelector(context, new AdaptiveTrackSelection.Factory());
+    final BandwidthMeter bandwidthMeter = new DefaultBandwidthMeter.Builder(context).setInitialBitrateEstimate(C.NETWORK_TYPE_UNKNOWN, 1_500_000).build();
+    final TrackSelector trackSelector = new DefaultTrackSelector(context, new AdaptiveTrackSelection.Factory(500, 2_000, 25_000, 0.6f));
```

Comment:
Do you mind adding some comments about the values that are used here? 🤔 
I know that some IDEs' features would enable seeing what these values stand for, but reading it on the web or in some editors without IntelliSense/function-signatures-resolution would cause problems in understanding.

Reply:
I had suggested these values based on my testing with different bandwidths. This particular setup was sufficient for videos to stream without buffering with internet speeds as low as 180kbps.  Although I'm sure these could probably be further improved.

`    public static final int DEFAULT_MIN_DURATION_FOR_QUALITY_INCREASE_MS = 500; // default: 10000
    public static final int DEFAULT_MAX_DURATION_FOR_QUALITY_DECREASE_MS = 2000; // default: 25000
    public static final int DEFAULT_MIN_DURATION_TO_RETAIN_AFTER_DISCARD_MS = 25000; // default: 25000
    public static final float DEFAULT_BANDWIDTH_FRACTION = 0.6f;`  //default 0.75

The explanation for these variables is available on https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/trackselection/AdaptiveTrackSelection.html

Hope this fix gets shipped in Expo 45 🤞